.github/ISSUE_TEMPLATE/enhancement.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Enhancement
+about: Suggest an idea for this project
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
```

Comment:
This is the default github feature request template which is actually quite clear and again really tries to guide the user:

```
**Is your feature request related to a problem? Please describe.**
A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]

**Describe the solution you'd like**
A clear and concise description of what you want to happen.

**Describe alternatives you've considered**
A clear and concise description of any alternative solutions or features you've considered.
```

**Is your feature request related to a problem? Please describe.**
A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]

**Describe the solution you'd like**
A clear and concise description of what you want to happen.

**Describe alternatives you've considered**
A clear and concise description of any alternative solutions or features you've considered.

Reply:
This is actually what I initially proposed, see my first commit: https://raw.githubusercontent.com/inmanta/inmanta/156f535111d742230e192df01e62fc7ed15c882a/.github/ISSUE_TEMPLATE/enhancement.md

(Except for the additional context related part)

Reply:
ok